static/app/constants/index.tsx

```diff
@@ -230,6 +237,66 @@ export const DATA_CATEGORY_NAMES = {
   [DataCategory.REPLAYS]: t('Session Replays'),
 };
 
+// https://github.com/getsentry/relay/blob/master/relay-common/src/constants.rs
+export const DATA_CATEGORY_INFO: Record<DataCategoryExact, DataCategoryInfo> = {
+  [DataCategoryExact.ERROR]: {
+    name: 'error',
+    apiName: 'error',
+    plural: 'errors',
+    displayName: 'error',
+    titleName: t('Errors'),
+    uid: 1,
+  },
+  [DataCategoryExact.TRANSACTION]: {
+    name: 'transaction',
+    apiName: 'transaction',
+    plural: 'transactions',
+    displayName: 'transaction',
+    titleName: t('Transactions'),
+    uid: 2,
+  },
+  [DataCategoryExact.ATTACHMENT]: {
+    name: 'attachment',
+    apiName: 'attachment',
+    plural: 'attachments',
+    displayName: 'attachment',
+    titleName: t('Attachments'),
+    uid: 4,
+  },
+  [DataCategoryExact.PROFILE]: {
+    name: 'profile',
+    apiName: 'profile',
+    plural: 'profiles',
+    displayName: 'profile',
+    titleName: t('Profiles'),
+    uid: 6,
+  },
+  [DataCategoryExact.REPLAY]: {
+    name: 'replay',
+    apiName: 'replay',
+    plural: 'replays',
+    displayName: 'replay',
+    titleName: t('Session Replays'),
+    uid: 7,
+  },
+  [DataCategoryExact.TRANSACTION_PROCESSED]: {
+    name: 'transaction_processed',
+    apiName: 'transactions',
+    plural: 'transactions',
+    displayName: 'transaction',
+    titleName: t('Transactions'),
+    uid: 8,
+  },
```

Comment:
Wasn't too sure about these names, but referred to this:
https://github.com/getsentry/getsentry/blob/master/getsentry/billing/utils/data_category.py#L54-L75